src/main/scala/lenthall/validation/ErrorOr.scala

```diff
@@ -19,4 +20,122 @@ object ErrorOr {
+    def flatMapN[B](f_19: (A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18) => ErrorOr[B]): ErrorOr[B] = (t19._1, t19._2, t19._3, t19._4, t19._5, t19._6, t19._7, t19._8, t19._9, t19._10, t19._11, t19._12, t19._13, t19._14, t19._15, t19._16, t19._17, t19._18, t19._19).tupled flatMap { case ((a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, ruchi, s)) => f_19(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, ruchi, s) }
+  }
+
+  implicit class ShortCircuitingFlatMapTuple20[A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19](val t20: Tuple20[ErrorOr[A0], ErrorOr[A1], ErrorOr[A2], ErrorOr[A3], ErrorOr[A4], ErrorOr[A5], ErrorOr[A6], ErrorOr[A7], ErrorOr[A8], ErrorOr[A9], ErrorOr[A10], ErrorOr[A11], ErrorOr[A12], ErrorOr[A13], ErrorOr[A14], ErrorOr[A15], ErrorOr[A16], ErrorOr[A17], ErrorOr[A18], ErrorOr[A19]]) extends AnyVal {
+    def flatMapN[B](f_20: (A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19) => ErrorOr[B]): ErrorOr[B] = (t20._1, t20._2, t20._3, t20._4, t20._5, t20._6, t20._7, t20._8, t20._9, t20._10, t20._11, t20._12, t20._13, t20._14, t20._15, t20._16, t20._17, t20._18, t20._19, t20._20).tupled flatMap { case ((a, b, c, d, e, f, g, h, i, jeff, k, l, m, n, o, p, q, r, s, t)) => f_20(a, b, c, d, e, f, g, h, i, jeff, k, l, m, n, o, p, q, r, s, t) }
+  }
+
+  implicit class ShortCircuitingFlatMapTuple21[A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20](val t21: Tuple21[ErrorOr[A0], ErrorOr[A1], ErrorOr[A2], ErrorOr[A3], ErrorOr[A4], ErrorOr[A5], ErrorOr[A6], ErrorOr[A7], ErrorOr[A8], ErrorOr[A9], ErrorOr[A10], ErrorOr[A11], ErrorOr[A12], ErrorOr[A13], ErrorOr[A14], ErrorOr[A15], ErrorOr[A16], ErrorOr[A17], ErrorOr[A18], ErrorOr[A19], ErrorOr[A20]]) extends AnyVal {
+    def flatMapN[B](f_21: (A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20) => ErrorOr[B]): ErrorOr[B] = (t21._1, t21._2, t21._3, t21._4, t21._5, t21._6, t21._7, t21._8, t21._9, t21._10, t21._11, t21._12, t21._13, t21._14, t21._15, t21._16, t21._17, t21._18, t21._19, t21._20, t21._21).tupled flatMap { case ((a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, thibault, u)) => f_21(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, thibault, u) }
+  }
+
+  implicit class ShortCircuitingFlatMapTuple22[A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20, A21](val t22: Tuple22[ErrorOr[A0], ErrorOr[A1], ErrorOr[A2], ErrorOr[A3], ErrorOr[A4], ErrorOr[A5], ErrorOr[A6], ErrorOr[A7], ErrorOr[A8], ErrorOr[A9], ErrorOr[A10], ErrorOr[A11], ErrorOr[A12], ErrorOr[A13], ErrorOr[A14], ErrorOr[A15], ErrorOr[A16], ErrorOr[A17], ErrorOr[A18], ErrorOr[A19], ErrorOr[A20], ErrorOr[A21]]) extends AnyVal {
```

Comment:
letters of the alphabet rather than `A0` .. `A21` would shorten this up (return type can be something two chars)